src/styles/bar.css

```diff
@@ -5,7 +5,22 @@ body {
   height: 100vh;
 }
 
+
 [role=application] {
   height: initial;
   flex: 1 100vh;
 }
+
+
+[role=banner] {
+  --silver: #d6d8da;
```

Comment:
@GoOz you're right, my mistake, we'll pass through a postcss plugin to handle them 😉